.gitattributes

```diff
@@ -2,3 +2,7 @@
 # decides that the content is text, its line endings are converted to LF on
 # checkin. When the file has been committed with CRLF, no conversion is done.
 * text=auto
+
+*.ml linguist-generated
```

Comment:
maybe leave a comment here to https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github to explain what this is doing

Reply:
This is a quirk of the fact that we are using Reason for everything other than autogenerated stuff, so maybe better to explicitly list the auto-generated files/directories -- Init.ml and the exercises directory I think are the culprits?